packages/graphql-mini-transforms/src/tests/webpack.test.ts

```diff
@@ -250,12 +250,31 @@ describe('graphql-mini-transforms/webpack', () => {
     it('has an undefined name when there are no named operations', async () => {
       const result = await extractDocumentExport(
         `query { shop { id } }`,
-        createLoaderContext({query: {simple: true}}),
+        createLoaderContext({query: {format: 'simple'}}),
       );
 
       expect(result).not.toHaveProperty('name');
     });
   });
+
+  describe('simple-persisted', () => {
+    it('provides a simple representation of the GraphQL document, with the source removed', async () => {
+      const simpleResult = await extractDocumentExport(
+        `query Shop { shop { id } }`,
+        createLoaderContext({query: {format: 'simple'}}),
+      );
+      const persistedResult = await extractDocumentExport(
+        `query Shop { shop { id } }`,
+        createLoaderContext({query: {format: 'simple-persisted'}}),
+      );
+
+      const expected = {...simpleResult};
+      delete expected.source;
+
+      expect(persistedResult).not.toHaveProperty('source');
```

Comment:
This should fix the failing test:

```suggestion
      const expected = {...simpleResult, source: ''};
```